src/ATen/native/xpu/SpectralOps.cpp

```diff
@@ -0,0 +1,32 @@
+#include <ATen/ATen.h>
+#include <ATen/native/Resize.h>
+#include <ATen/native/xpu/mkl/SpectralOps.h>
+#include <ATen/xpu/XPUNativeFunctions.h>
+
+namespace at {
+
+Tensor XPUNativeFunctions::_fft_c2c(
+    const Tensor& self,
+    IntArrayRef dim,
+    int64_t normalization,
+    bool forward) {
+  TORCH_CHECK(self.is_complex());
+
+  return native::xpu::_fft_c2c_mkl(self, dim, normalization, forward);
+}
+
+Tensor& XPUNativeFunctions::_fft_c2c_out(
+    const Tensor& self,
+    IntArrayRef dim,
+    int64_t normalization,
+    bool forward,
+    Tensor& out) {
+  TORCH_CHECK(self.is_complex());
+
+  Tensor result = native::xpu::_fft_c2c_mkl(self, dim, normalization, forward);
+  at::native::resize_output(out, result.sizes());
+  out.copy_(result);
+  return out;
```

Comment:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/d0185f08-67<img width="716" alt="image" src="https://github.com/user-attachments/assets/d0185f08-6774-403b-8979-63ad569c0441">
Is `_fft_apply_normalization_out` a step specific for CUDA cufft implementation?74-403b-8979-63ad569c0441">
Is `_fft_apply_normalization_out` a step specific for CUDA cufft implementation?